command/genesis/genesis.go

```diff
@@ -125,6 +125,14 @@ func setFlags(cmd *cobra.Command) {
 			"the type of validators in IBFT",
 		)
 
+		cmd.Flags().StringVar(
+			&params.validatorRootPath,
```

Comment:
Should we validate if the path exists and stop the execution otherwise?

Reply:
It is done already: https://github.com/0xPolygon/polygon-edge/blob/develop/command/genesis/params.go#L206-L210